test/countNumberOfLowercase.test.js

```diff
@@ -0,0 +1,27 @@
+import { equal } from 'assert';
+import { typeOfArgumentErrorMessage, countNumberOfLowercase } from '../countNumberOfLowercase.js';
+
+it('Mix of lowercase letters, uppercase letters, numbers and symbols in a string', () => {
+    equal(countNumberOfLowercase('abcABC123!@€£#$%^&*()_-+=}{[]|\':;?/>.<,~'), 3);
+})
+
+it('Just lowercase letters in a string', () => {
+    equal(countNumberOfLowercase('zxcpo'), 5);
+})
+
+it('No lowercase letters in a string', () => {
+    equal(countNumberOfLowercase('ABC123!%^&*()_-+=|'), 0);
+})
+
+it('Empty string', () => {
+    equal(countNumberOfLowercase(''), 0);
+})
+
+it('Should throw an Error when the type of argument is not a String', () => {
+    try {
+        countNumberOfLowercase([123, 1])
```

Comment:
[optional improvement] for readability, you could do something like 
`["thisIsArray"]`

Reply:
```suggestion
        countNumberOfLowercase(["thisIsArray"])
```

Did you know you could do this as part of the PR? 
Called `suggestion`:
![image](https://github.com/AnnaSanna/CodeWarsPlayGround/assets/4237268/463fe074-873b-43ee-8432-d6af9fd382bb)

Reply:
need to discuss